File: VB/MultipleORMsExample.Module/Module.vb
Imports System.Data.Entity
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating
Imports DevExpress.ExpressApp.EF
Imports MultipleORMsExample.Module.BusinessObjects

Namespace MultipleORMsExample.Module
    Public NotInheritable Partial Class MultipleORMsExampleModule
        Inherits ModuleBase

        Public Sub New()
            InitializeComponent()
            Database.SetInitializer(Of MyDbContext)(New DropCreateDatabaseIfModelChanges(Of MyDbContext)())
        End Sub
        Public Overrides Function GetModuleUpdaters(ByVal objectSpace As IObjectSpace, ByVal versionFromDB As Version) As IEnumerable(Of ModuleUpdater)
            Dim updater As ModuleUpdater = New DatabaseUpdate.Updater(objectSpace, versionFromDB)
            Return New ModuleUpdater() { updater }
        End Function
    End Class
End Namespace

File: CS/MultipleORMsExample.Module/Module.cs
using System;
using System.Collections.Generic;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;
using DevExpress.ExpressApp.EF;
using MultipleORMsExample.Module.BusinessObjects;
using System.Data.Entity;


namespace MultipleORMsExample.Module {
    public sealed partial class MultipleORMsExampleModule : ModuleBase {
        public MultipleORMsExampleModule() {
            InitializeComponent();
            ExportedTypeHelpers.AddExportedTypeHelper(new EFExportedTypeHelperCF());
            Database.SetInitializer<MyDbContext>(new DropCreateDatabaseIfModelChanges<MyDbContext>());
        }
        public override IEnumerable<ModuleUpdater> GetModuleUpdaters(IObjectSpace objectSpace, Version versionFromDB) {
            ModuleUpdater updater = new DatabaseUpdate.Updater(objectSpace, versionFromDB);
            return new ModuleUpdater[] { updater };
        }
    }
}
